config/config.go

```diff
@@ -178,16 +179,18 @@ type TLSClientSpec struct {
 	AllowUnverifiedSSLCert bool   `yaml:"allow_unverified_ssl_cert,omitempty" mapstructure:"allow_unverified_ssl_cert,omitempty"`
 }
 
-// Tenant is config relating to an Apigee tentant
+// Tenant is config relating to an Apigee tenant
 type Tenant struct {
-	InternalAPI         string          `yaml:"internal_api,omitempty" mapstructure:"internal_api,omitempty"`
-	RemoteServiceAPI    string          `yaml:"remote_service_api" mapstructure:"remote_service_api"`
-	OrgName             string          `yaml:"org_name" mapstructure:"org_name"`
-	EnvName             string          `yaml:"env_name" mapstructure:"env_name"`
-	Key                 string          `yaml:"key,omitempty" mapstructure:"key,omitempty"`
-	Secret              string          `yaml:"secret,omitempty" mapstructure:"secret,omitempty"`
-	ClientTimeout       time.Duration   `yaml:"client_timeout,omitempty" mapstructure:"client_timeout,omitempty"`
-	TLS                 TLSClientSpec   `yaml:"tls,omitempty" mapstructure:"tls,omitempty"`
+	InternalAPI      string        `yaml:"internal_api,omitempty" mapstructure:"internal_api,omitempty"`
+	RemoteServiceAPI string        `yaml:"remote_service_api" mapstructure:"remote_service_api"`
+	OrgName          string        `yaml:"org_name" mapstructure:"org_name"`
+	EnvName          string        `yaml:"env_name" mapstructure:"env_name"`
+	Key              string        `yaml:"key,omitempty" mapstructure:"key,omitempty"`
+	Secret           string        `yaml:"secret,omitempty" mapstructure:"secret,omitempty"`
+	ClientTimeout    time.Duration `yaml:"client_timeout,omitempty" mapstructure:"client_timeout,omitempty"`
+	TLS              TLSClientSpec `yaml:"tls,omitempty" mapstructure:"tls,omitempty"`
+	// OperationConfigType set to "proxy" switches to Apigee "proxy" type API Product operations from "remoteservice" type
+	OperationConfigType string          `yaml:"operationConfigType,omitempty" mapstructure:"operationConfigType,omitempty"`
```

Comment:
`operation_config_type` in the yaml and mapstructure tag to be consistent with others.